modules/sdk/src/main/scala/org/tessellation/sdk/http/p2p/clients/L0TrustClient.scala

```diff
@@ -14,16 +15,21 @@ import org.http4s.client.Client
 trait L0TrustClient[F[_]] {
   def getCurrentTrust: PeerResponse[F, TrustScores]
   def getPreviousTrust: PeerResponse[F, OrdinalTrustMap]
+
+  def getPreviousSeedlistPeerLabels: PeerResponse[F, TrustLabels]
 }
 
 object L0TrustClient {
-  def make[F[_]: Async: SecurityProvider](client: Client[F]): L0TrustClient[F] =
+  def make[F[_]: Async: SecurityProvider: KryoSerializer](client: Client[F]): L0TrustClient[F] =
     new L0TrustClient[F] {
 
       def getCurrentTrust: PeerResponse[F, TrustScores] =
         PeerResponse[F, TrustScores]("trust/current")(client)
 
       def getPreviousTrust: PeerResponse[F, OrdinalTrustMap] =
         PeerResponse[F, OrdinalTrustMap]("trust/previous")(client)
+
+      def getPreviousSeedlistPeerLabels: PeerResponse[F, TrustLabels] =
```

Comment:
what does `seedlist` mean in this context? shouldn't it be just `getPreviousPeerLabels` as this is the endpoint you query for?

Reply:
Whenever peers update their deterministic trust map, they gossip it out as `SnapshotOrdinalPublicTrust`s. In other words, they gossip out a `Map[PeerId, Double]`, which is a mapping of how trustworthy they perceive other peers to be.

Nodes take these gossiped `PublicTrust`s and store them, associated to the peer that sent them, in their deterministic trust map as peerLabels. What this endpoint returns is:

Starting with the peerLabels, we grab the `labels` from the `PublicTrust` for each seedlist peer entry. In other words, we only use `PublicTrust`s from the node's seedlist peers. We then weight the trust values of each `labels` by how much we trust the peer that sent the `PublicTrust`. So we have an `Iterable` of weighted trust maps from each seedlist peer.

Then we combine the trust maps by averaging the trust scores for each entry. So, we are left with a trust map containing all the `PeerId`s gossiped to us by our seedlist peers during the last deterministic trust map update cycle, as well as their averaged, weighted trust score.

The seedlist peers are the peers designated upon node start up as its seedlist. The trustworthiness of each seedlist peer is loaded the same way. This functionality was added in PR https://github.com/Constellation-Labs/tessellation/pull/666.

Reply:
Ok, I see. Nevertheless, you query for previous peer labels, so IMO it's how the client should be named like. The info about the seedlist is imo the implementation detail that should be hidden behind the API call.

Reply:
Renamed the function to `getPreviousPeerLabels`.